pkg/dnfjson/dnfjson.go

```diff
@@ -205,7 +205,7 @@ func (s *Solver) Depsolve(pkgSets []rpmmd.PackageSet) ([]rpmmd.PackageSpec, []rp
 
 	output, err := run(s.dnfJsonCmd, req)
 	if err != nil {
-		return nil, nil, err
+		return nil, nil, fmt.Errorf("running osbuild-depsolve-dnf failed:\n%w", err)
```

Comment:
Why is there an extra `\n` but in the others changes there is not?

Reply:
Hmm, maybe I saw that in this case `err` is larger and this looks nicer in the end - but I can also make this symmetric with the rest

Reply:
I don't care too much, I approved the PR. I personally would still make it symetrical unless there is a good reason not to (having a test that contains a real world example might be nice way to check/document what the error does look like). But I approved, those are details.